CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+
+## [0.9.4-beta] - 2016-04-20
+### Added
+- Deploy to App Engine flexible environment tools menu item. ([#635](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/635))
+- Support for Maven based projects as deployment sources for App Engine flexible environment deployments ([#600](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/600))
+
+### Changed
+- App Engine flexible environment deployment can be cancelled by disconnecting to our App Engine applicaiton server. ([#581](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/581))
```

Comment:
fixed.

On Mon, May 2, 2016 at 11:01 AM, Etan Shaul <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/668#discussion_r61750319>
> :
>
> > @@ -0,0 +1,18 @@
> > +# Change Log
> > +All notable changes to this project will be documented in this file.
> > +
> > +## [Unreleased]
> > +
> > +
> > +## [0.9.4-beta] - 2016-04-20
> > +### Added
> > +- Deploy to App Engine flexible environment tools menu item. ([#635](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/635))
> > +- Support for Maven based projects as deployment sources for App Engine flexible environment deployments ([#600](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/600))
> > +
> > +### Changed
> > +- App Engine flexible environment deployment can be cancelled by disconnecting to our App Engine applicaiton server. ([#581](https://github.com/GoogleCloudPlatform/gcloud-intellij/issues/581))
>
> "application" typo. LGTM.
>
> Do you think it makes sense for individual committers to gradually add to
> this (for non-trivial changes)? The alternative is to compile them all at
> once at the end.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/668/files/04b02fffca09c4a43dc7250df90cbeebcea29395#r61750319>
>